docs/howto/account-billing/Delete-account.md

```diff
@@ -0,0 +1,24 @@
+# Deleting your account
+
+In case you decide to close your account in {{brand}}, you may do so via the [{{gui}}](https://{{gui_domain}}).
+Log into your account and click on the profile button at the top right.
+
+![Orange avatar person with white background.](assets/picture-profile.png)
+
+A pane named _Account settings_ will slide over from the right-hand side of the browser window. To delete your account, click the _Settings_ tab.
+
+![A Settings tab with a red box around it](assets/accountsettings.png)
+
+Scroll to the bottom of the pane, and click the red button named _Delete Account_.
+
+![Black text on white background with info what will happen when you click on the delete button](assets/Delete-account-picture.png)
+
+Choose a reason for leaving and type in your current password. Click the red button named _Delete Account_. That will create a request for account deletion.
```

Comment:
Since this is the second time we click on the red button, I suggest the following _slight_ change:

```suggestion
Choose a reason for leaving and type in your current password. Click the red button named _Delete Account_ one more time. That will create a request for account deletion.
```